src/debian/snmp_smartctl.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+/usr/sbin/smartctl --scan | grep -E "^/dev/sd" | awk '{ print $1 }' | while read i; do
```

Comment:
Also look for nvme device: `grep -E '^/dev/(sd|nvme)'`.

Reply:
ok